api-docs/swagger.yml

```diff
@@ -263,53 +276,160 @@ paths:
           description: Healthy
         '500':
           description: Not healthy
+securityDefinitions:
+  basicAuth:
+    type: basic
 definitions:
+  Info:
+    type: object
```

Comment:
This may have changed in 0.8.9, please can you double-check?

Reply:
It changed. I modified it in 5c657e6